docs/readmes/lte/debug_dp_probe.md

```diff
@@ -16,19 +16,22 @@ On your GW host, run the following command
 
 For example:
 ```sh
-$ dp_probe_cli.py -i 1010000051013 --direction DL
-IMSI: 1010000051013, IP: 192.168.128.13
-Running: sudo ovs-appctl ofproto/trace gtp_br0 tcp,in_port=local,ip_dst=192.168.128.13,ip_src=8.8.8.8,tcp_src=80,tcp_dst=3372
-Datapath Actions: set(tunnel(tun_id=0x1000008,dst=10.0.2.241,ttl=64,tp_dst=2152,flags(df|key))),pop_eth,2
-Downlink rules: allowlist_sid-IMSI1010000051013-magma.ipv4
+dp_probe_cli.py -i 001010000000004 --direction DL
+IMSI: 001010000000004, IP: 192.168.128.15
+LOCAL: n_packets=333, n_bytes=21998
+mtr0: n_packets=0, n_bytes=0
+Running: sudo ovs-appctl ofproto/trace gtp_br0 tcp,in_port=local,ip_dst=192.168.128.15,ip_src=8.8.8.8,tcp_src=80,tcp_dst=3372
+Datapath Actions: set(tunnel(tun_id=0xa000428,dst=192.168.60.141,ttl=64,tp_dst=2152,flags(df|key))),pop_eth,set(skb_mark(0x11)),2
+Downlink rules: allowlist_sid-IMSI001010000000004-magma.ipv4
```

Comment:
can you add separate option just to print the stats for UL or DL

Reply:
@pshelar  thanks for checking! I updated the PR. Let me know if you have any other comments